src/table/virtualized-table/Body.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { memo, useMemo, useLayoutEffect, useRef, useImperativeHandle, forwardRef } from 'react';
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
Remove this and find a better solution. See other comment about a possible solution.